repos/token.txt

```diff
@@ -0,0 +1 @@
+input_your_github_token
```

Comment:
I think we should provide, how to get a token, and what permissions you need to set for it.